src/clp_ffi_js/ir/StreamReader.cpp

```diff
@@ -117,8 +117,12 @@ EMSCRIPTEN_BINDINGS(ClpStreamReader) {
     // JS types used as inputs
     emscripten::register_type<clp_ffi_js::ir::DataArrayTsType>("Uint8Array");
     emscripten::register_type<clp_ffi_js::ir::LogLevelFilterTsType>("number[] | null");
+    emscripten::register_type<clp_ffi_js::ir::ReaderOptions>("{timestampKey: string} | null");
 
     // JS types used as outputs
+    emscripten::enum_<clp_ffi_js::ir::StreamType>("IrStreamType")
```

Comment:
Just double-checking, the TS type should remain `IrStreamType`, right?

Reply:
Right, I keep the `Ir` prefix intentionally.

To be precise, types registered via this method are not TS types. For every enum, emscripten creates an object, which is comparable with any transpiled enum value. For example, `Module.IrStreamType.STRUCTURED` / `Module.IrStreamType.UNSTRUCTURED` can be used to check against the return values of `reader.getIrStreamType()` in our case.

Reply:
Reference: https://www.github.com/emscripten-core/emscripten/issues/18585